sdk/aqueduct/decorator.py

```diff
@@ -18,10 +18,13 @@
     artifact_name_from_op_name,
 )
 from aqueduct.error import AqueductError
+from pandas import DataFrame
 
 # Valid inputs and outputs to our operators.
 OutputArtifact = Union[TableArtifact, MetricArtifact, CheckArtifact]
 InputArtifact = Union[TableArtifact, MetricArtifact, ParamArtifact]
+InputArtifact_list = [TableArtifact, MetricArtifact, ParamArtifact]
+InputArtifact_local = Union[TableArtifact, MetricArtifact, ParamArtifact,DataFrame]
```

Comment:
These variables are a mix of camelcase and underscores. Lets just do camelcase for types in python. Underscores for everything else in python.

Reply:
Also, `it needs to be `InputArtifactList = List[TableArtifact, MetricArtifact...]`. Mypy would catch this once I get that in.

Reply:
Just fix the issues! Since "a list is a homogenous structure containing values of one type", I can't necessarily use a List to wrap three types. Therefore, I turn to a solution on StackOverflow. Let me know if you think that works or not.
https://stackoverflow.com/questions/45957615/check-a-variable-against-union-type-at-runtime-in-python-3-6